src/pages/Add.js

```diff
@@ -1,10 +1,7 @@
-import AddForm from "../components/AddForm";
-
 export default function Add() {
   return (
     <section>
       <h2 className="Heading">add a new memory!</h2>
-      <AddForm />
```

Comment:
Why are you removing this here?

Reply:
Ah i see you created a form when you double click and add a new pin. Looks good, but then you don't need the create new marker section in your navigation

Reply:
i will remove the add new marker section! if i have the time ill make it a fav page :)